### PR TITLE
Update forum references

### PR DESCRIPTION
**Pull request by aelamrani**

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -5,9 +5,9 @@ Here are few guidelines that should help you get started.
 
 == Using GitHub issues
 
-We use GitHub issues to track bugs and enhancements. Found a bug in the source code? You want to propose new features or enhancements  You can help us by submitting an issue in our https://github.com/gravitee-io/json-schema-generator-maven-plugin[repository]. Before you submit your issue, search the https://github.com/gravitee-io/json-schema-generator-maven-plugin/issues[issues archive] or the https://waffle.io/gravitee-io/release[backlog]; maybe your question was already answered.
```
Comment (jhaeyaert) [inline CONTRIBUTING.adoc:8]: The issues are not centralized anymore ? https://github.com/gravitee-io/json-schema-generator-maven-plugin/issues

**Review by jhaeyaert — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.adoc (hunk 1 above)
Comment (aelamrani) [reply CONTRIBUTING.adoc:8]: yes it is, not sure to understand your comment

Comment (jhaeyaert) [reply CONTRIBUTING.adoc:8]: Issues link should redirect to https://github.com/gravitee-io/issues/issues no ?

Comment (aelamrani) [reply CONTRIBUTING.adoc:8]: oh ok I did not change this, I can do it

Comment (aelamrani) [reply CONTRIBUTING.adoc:8]: fixed

**Pull request merged**